src/Features/LanguageServer/Protocol/Handler/MethodAttribute.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
 [MetadataAttribute]
 internal class MethodAttribute : LanguageServerEndpointAttribute
 {
-    public MethodAttribute(string method) : base(method)
+    public MethodAttribute(string method, string? language = null) : base(method, language)
```

Comment:
make non-optional.

Reply:
In order to make it non-optional we have to allow this to specify all possible languages we support, and export all roslyn handlers with c#/vb/f# etc. 

Which is probably fine.  But will be a large change.

Reply:
Thought about this some more - and I posted a comment somewhere else I can't find .  I think we can make this non-optional - just have this MethodAttribute be exported for C#/F#/VB explicitly, and add a new one for XAML.

We can rename this attribute in a separate PR (to avoid adding even more changes here).

Reply:
Done